manifests/t/Tormak/SteamArtManager/2.5.0/Tormak.SteamArtManager.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Tormak.SteamArtManager
+PackageVersion: 2.5.0
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Valve.Steam
Installers:
```

Reply:
Since a user needs to have set up steam and own games for the program to serve any real purpose, I think having a dialog that says "you must have steam installed" is sufficient rather then install a dependency. thoughts?